easybuild/easyblocks/m/mvapich2.py

```diff
@@ -70,12 +70,24 @@ def configure_step(self):
 
         # enable specific support options (if desired)
         if self.cfg['withmpe']:
-            add_configopts.append('--enable-mpe')
+            # --enable-mpe is coming from MPICH.
```

Comment:
I'm not a big fan of including two versions that need to be kept in sync in all the MVAPICH2 easyconfigs...

My remark was actually purely on the comment itself, not on the version check for `--enable-mpe` below... That's good as is imho.

As it is right now, the MPICH easyblock doesn't support a way to include `--enable-mpe`, so right now, it's a non-issue?